playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -45,6 +46,59 @@
   - set_fact:
       pre_upgrade_complete: True
 
+- import_playbook: label_nodes.yml
+
+# To upgrade, we need masters to be capable of signing certificates
+- hosts: oo_masters
+  serial: 1
+  tasks:
+  - name: Enable core bootstrapping components
+    include_tasks: ../../../../openshift-master/private/tasks/enable_bootstrap.yml
+  - name: Place shim commands on the masters before we begin the upgrade
+    import_role:
+      name: openshift_control_plane
+      tasks_from: static_shim
+
+# TODO: need to verify settings about the bootstrap configs
+# 1. Does network policy match the master config
+
+- hosts: oo_all_hosts
```

Comment:
Don't do anything against oo_all_hosts during control plane upgrades.  Don't do anything against nodes during control plane upgrades.

Reply:
We’re changing a control plane dynamic component that affects all nodes (like daemonsets).  This particular step may no longer be necessary since I added a ratchet to dynamic config, which I’ll verify.

Reply:
I'll test without it, I agree in general although the model with daemonsets won't really match that.  Control plane components will upgrade, and the components will have to tolerate older nodes.  We obviously don't want to disrupt workloads, but upgrading the control plane will affect nodes (sdn, registry, router, config sync, bootstrapping etc all have impacts).

This was intended to pause dynamic reconfiguration until nodes reestablished it.